aws-iot-thing/aws-iot-thing.json

```diff
@@ -0,0 +1,80 @@
+{
+  "typeName": "AWS::IoT::Thing",
+  "description": "Resource Type definition for AWS::IoT::Thing",
+  "additionalProperties": false,
+  "properties": {
+    "Id": {
+      "type": "string"
+    },
+    "Arn": {
+      "type": "string"
+    },
+    "AttributePayload": {
+      "$ref": "#/definitions/AttributePayload"
+    },
+    "ThingName": {
+      "type": "string",
+      "pattern": "[a-zA-Z0-9:_-]+",
+      "minLength": 1,
+      "maxLength": 128
+    }
+  },
```

Comment:
Would we need to include thing type name and billing group name here? https://paste.amazon.com/show/aarongup/1643746470

Reply:
This will be added as a follow up after current release.